src/app/content/practiceQuestions/components/Question.tsx

```diff
@@ -24,12 +24,15 @@ export const QuestionWrapper = styled.form`
 `;
 
 // tslint:disable-next-line: variable-name
-export const QuestionContent = styled(ContentExcerpt)`
+export const QuestionContent = styled(React.forwardRef((props, ref) => <div tabIndex={0}>
+  <ContentExcerpt {...props} tabIndex={-1} ref={ref} />
+</div>))`
```

Comment:
I'm not sure why these changes are in this or?

Reply:
I did something weird while interactive rebasing lol don't worry about this, I'll remove them

Reply:
I did something weird while interactive rebasing lol don't worry about this file, I'll remove it